server/api/googleLogin.js

```diff
@@ -43,35 +42,21 @@ async function googleLogin(req, res, next) {
   );
 }
 
-async function testGoogleLogin(req, res, next) {
-  const username = req.body.googleIdToken;
-  await googleSignup(req, res, username);
-}
-
 async function googleSignup(req, res, username) {
-  if (!username ) {
-    res.status(400).send("Error: Username not set!");
-    return;
-  }
```

Comment:
note: this check is always false as the requirement is checked at the beginning of the callback